client/jetpack-dashboard/sidebar.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { memoize } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Sidebar from 'layout/sidebar';
+import SidebarItem from 'layout/sidebar/item';
+import ExpandableSidebarMenu from 'layout/sidebar/expandable';
+import getCurrentRoute from 'state/selectors/get-current-route';
+
+class JetpackDashboardSidebar extends React.PureComponent {
+	isItemSelected( itemPath, isStrict = true ) {
+		const { path } = this.props;
+
+		if ( isStrict ) {
+			return path === itemPath;
+		}
+
+		return path.indexOf( itemPath ) === 0;
+	}
+
+	toggleSection = memoize( id => () => this.props.toggleSection( id ) );
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<Sidebar>
+				<SidebarItem
+					label={ translate( 'Jetpack.com Dashboard' ) }
+					link="/"
+					selected={ this.isItemSelected( '/' ) }
+					icon="plans"
+				/>
+				<ExpandableSidebarMenu expanded title={ translate( 'Security' ) } icon="lock">
+					<SidebarItem
+						label={ translate( 'Dashboard' ) }
+						link="/security"
+						selected={ this.isItemSelected( '/security' ) }
+					/>
+					<SidebarItem
+						label={ translate( 'Backups' ) }
+						link="/security/backups"
+						selected={ this.isItemSelected( '/security/backups' ) }
+					/>
+					<SidebarItem
+						label={ translate( 'Malware scan' ) }
+						link="/security/scan"
+						selected={ this.isItemSelected( '/security/scan' ) }
+					/>
+					<SidebarItem
+						label={ translate( 'Anti-spam' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Akismet Anti-Spam' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/jetpack-dashboard-prototype)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
We're not planning to merge any UI bits for now.